docs/tutorials/benchmarking/README.md

```diff
@@ -0,0 +1,17 @@
+# Benchmarking in Syne Tune
+
+
+This tutorial provides an overview of tooling which facilitates benchmarking of
```

Comment:
should we explain what we mean by benchmarking first? to me (aka a first time user) it was not clear at first, and i had to ask.

Reply:
Also lets explain different type of benchmarking: tabulated, surrogate, and real. 
we should explain these before getting into benchmarking with different backends

Reply:
I'll add a link to the docs of the blackbox repo

Reply:
Added spoe explanation